myhpi/core/models.py

```diff
@@ -16,12 +16,14 @@
 class InformationPage(Page):
     body = MarkdownField()
     visible_for = ParentalManyToManyField(Group, related_name="visible_informationpages")
+    is_public = BooleanField()
```

Comment:
Why don't we add this Boolean to `Page` directly?

Reply:
Page is something that is taken from wagtail - might this be the point why it is added here?

Reply:
Very well! We could add our own `BasePage` then instead of duplicating code

Reply:
that sounds like a good idea, would also make the permission check easier (currently checks if it is in the list of our page types)

Reply:
I'm about to do that but it's hard to migrate when you add a new superclass that wasn't there previously. It will most likely result in a data loss upon migration.

Reply:
`BasePage` class added.